blueprints/addon/files/npmignore

```diff
@@ -21,6 +21,7 @@
 /.watchmanconfig
 /CONTRIBUTING.md
 /ember-cli-build.js
+/eslint.config.mjs
```

Comment:
These can go?

```
/.eslintignore
/.eslintrc.js
```

Reply:
Yeah, ignore is specified in the config, and the eslint config is named different now

Reply:
done